packages/epo-widget-lib/src/widgets/LightCurvePlot/A11Y/LightCurveLabel/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import minBy from "lodash/minBy";
+import { calculateResidual, formatMagnitudePoints } from "../../helpers";
+import { between } from "@/lib/utils";
+import * as Styled from "./styles";
+
+interface LightCurveLabelProps {
+  data: ReturnType<typeof formatMagnitudePoints>;
+  controlledById?: string;
+  estimatedPeak: number;
+  gaussianWidth: number;
+  yOffset: number;
+}
+
+const LightCurveLabel: FunctionComponent<LightCurveLabelProps> = ({
+  data,
+  gaussianWidth,
+  yOffset,
+  controlledById,
+  estimatedPeak,
+}) => {
+  const {
+    t,
+    i18n: { language },
+  } = useTranslation();
+
+  const meanResidual =
+    data
+      .filter(({ x }) => between(x, 0, 15))
+      .reduce(
+        (prev, curr) => prev + calculateResidual(curr, gaussianWidth, yOffset),
+        0
+      ) / data.length;
+  const { y: brightest } = minBy(data, ({ y }) => y) || {};
+
+  return (
+    <Styled.HiddenOutput form={controlledById}>
```

Comment:
this output is tied to the slider controls and reads out the residual to help screenreader users get their line closer.